app/views/groups/create/clients/confirm_remove_client.scala.html

```diff
@@ -34,13 +34,19 @@
     clientToRemove: DisplayClient,
     backLink: Call = routes.CreateGroupSelectClientsController.showReviewSelectedClients(None, None),
     formAction: Call = routes.CreateGroupSelectClientsController.submitConfirmRemoveClient,
+        paragraphKey: Option[String] = None,
 )(implicit request: Request[_], msgs: Messages, appConfig: AppConfig)
 
     @title = @{msgs("common.remove.client", clientToRemove.name)}
 
     @layout(title = title, formErrors = form.errors, backLinkHref = Some(backLink.url)) {
 
         @caption(msgs("common.caption.group.name", groupName))
+        @h1(title)
+
+        @if(paragraphKey.isDefined){
+            @p(paragraphKey.get)
+        }
 
         @formWithCSRF(action = formAction) {
             @radioButtons(
```

Comment:
You shouldn't leave the radio buttons without a legend. If it's not the h1 (since you already have the h1 above) I'd make it visually hidden.

Tbf, I'd do something like this instead

```
 @if(paragraphKey.isDefined){
    @h1(title)
    @p(paragraphKey.get)
}

val h1class= if(paragraphKey.isDefined){ "govuk-visually-hidden" } else {"govuk-heading-xl"}
... radios
legend = Some(LegendData(title, isH1 = !paragraphKey.isDefined, class=h1class)),
```